services/121-service/src/utils/formatDate.ts

```diff
@@ -0,0 +1,11 @@
+function padTo2Digits(num: number): string {
```

Comment:
Refactor: this same function is defined in 2 places in our code base, and the implementation looks identical.